src/test/run-make-fulldeps/pretty-expanded/input.rs

```diff
@@ -1,6 +1,6 @@
-#[crate_type="lib"]
-
-// #13544
+// FIXME: This should be replaced with some other derive macro
+#![feature(rustc_encodable_decodable)]
+#[crate_type="lib"] // Why is this an outer attribute?
 
 extern crate rustc_serialize;
```

Comment:
You can likely swap this test out for derive(Debug) which'll make it simpler and drop the soft-deprecated feature use.